db/migrate/20190314181829_migrate_open_registrations_setting.rb

```diff
@@ -0,0 +1,15 @@
+class MigrateOpenRegistrationsSetting < ActiveRecord::Migration[5.2]
+  def up
+    open_registrations = Setting.find_by(var: 'open_registrations')
+    return if open_registrations.nil? || open_registrations.value
+    setting = Setting.where(var: 'registrations_mode').first_or_initialize(var: 'registrations_mode')
+    setting.update(value: 'none')
```

Comment:
I suggest `approved`

Reply:
Why? It would be confusing for people that really wanted to close their registrations…

Reply:
I think a lot of closed registrations are due to the spammer problem, or uncontrolled growth. For example, that instance that only allowed people to sign up during the full moon... With this option, people can sign up any time, but only be allowed in during the full moon.

I fear that otherwise this feature will not be discovered by those who it would benefit :confused:

Reply:
I think some instances really want closed registrations.

I think the problem for instances that have closed with Instances that closed registrations due to spam problems or uncontrolled growth will just shift to approval requests. And it'll be worse on the admins if you disrupt their current ad-hoc approval flow without prior notice.